pkg/cryptoauthlib/psa_se_driver/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = psa_se_driver
```

Comment:
AFAIR the folder name is assigned by default, so you only need to set `MODULE` if you explicitly want a diverging module name

Reply:
Removed